src/accounts/superfluid.ts

```diff
@@ -0,0 +1,197 @@
+    }
+
+    private alephPerHourToFlowRate(alephPerHour: Decimal): ethers.BigNumber {
+        return this.alephToWei(alephPerHour).div(ethers.BigNumber.from(3600));
+    }
+
+    private flowRateToAlephPerHour(flowRate: ethers.BigNumber | string): Decimal {
+        return new Decimal(ethers.utils.formatEther(ethers.BigNumber.from(flowRate).mul(ethers.BigNumber.from(3600))));
+    }
+
+    /**
+     * Get the regular ALEPH balance of the account.
+     */
+    public async getALEPHBalance(): Promise<Decimal> {
+        // @todo: implement
+        return new Decimal(0);
+    }
+
+    public async getALEPHxBalance(): Promise<Decimal> {
+        if (!this.wallet) throw Error("PublicKey Error: No providers are set up");
+        if (!this.alephx) throw new Error("Account not initialized");
+        const balance = await this.alephx.balanceOf({ account: this.address, providerOrSigner: this.wallet.provider });
+        return this.weiToAleph(balance);
+    }
+
+    public async wrapALEPH(amount: Decimal): Promise<void> {
+        if (!this.wallet) throw Error("PublicKey Error: No providers are set up");
+        if (!this.alephx) throw new Error("Account not initialized");
+        const op = this.alephx.upgrade({
+            amount: this.alephToWei(amount).toString(),
+        });
+        await op.exec(this.wallet.provider.getSigner());
+    }
+
+    public async unwrapALEPHx(amount: Decimal): Promise<void> {
+        if (!this.wallet) throw Error("PublicKey Error: No providers are set up");
+        if (!this.alephx) throw new Error("Account not initialized");
+        const op = this.alephx.downgrade({
+            amount: this.alephToWei(amount).toString(),
+        });
+        await op.exec(this.wallet.provider.getSigner());
+    }
+
+    /**
+     * Get the ALEPHx flow rate to a given receiver in ALEPHx per hour.
+     * @param receiver The receiver address.
+     */
+    public async getALEPHxFlow(receiver: string): Promise<Decimal> {
+        if (!this.wallet) throw Error("PublicKey Error: No providers are set up");
+        if (!this.alephx) throw new Error("Account not initialized");
+        const flow = await this.alephx.getFlow({
+            sender: this.address,
+            receiver,
+            providerOrSigner: this.wallet.provider,
+        });
+        if (!flow) {
+            return new Decimal(0);
+        } else {
+            return this.flowRateToAlephPerHour(flow.flowRate);
+        }
+    }
+    public async getNetALEPHxFlow(): Promise<Decimal> {
+        if (!this.wallet) throw Error("PublicKey Error: No providers are set up");
+        if (!this.alephx) throw new Error("Account not initialized");
+        const flow = await this.alephx.getNetFlow({
+            account: this.address,
+            providerOrSigner: this.wallet.provider,
+        });
+        return this.flowRateToAlephPerHour(flow);
+    }
+
+    public async increaseALEPHxFlow(receiver: string, alephPerHour: Decimal): Promise<void> {
```

Comment:
I think that will be better to use the same format in all the places, as we use ALEPH/second on the backend, and Superfluid also uses seconds as unit for the flow rate, so maybe it's better to use it here by seconds.
Also, we have the `alephPerHourToFlowRate` and `flowRateToAlephPerHour` methods to make it easier on the frontend to convert to that measure.

Reply:
The helper methods are intended to convert `Decimals` to `ethers.BigNumbers`. As on the frontend the per-hour definition is usually used, I decided to go that way, after consulting with the other frontend devs on TG